README.md

```diff
@@ -0,0 +1,20 @@
+# libmapvxl
+A C library for handling the VXL map format. Used primarily in [SpadesX][spadesx].
+
+## Compilation
```

Comment:
Its a library thats supposed to be directly compiled by another software. There is no need to compile it on its own.

Reply:
Being able to compile it on it's own is the whole point of a library, that way it can be shared between software's like an AoS client that might want to use your library. It's also needed to exclusively work on the library like I intend on doing. If you want I can rename the section Independent compilation?

Reply:
Its fine